### PR TITLE
Add null model rationale

### PR DESCRIPTION
**Pull request by hyanwong**
Suggested by Per.

### CODE REVIEW

**Review by jeromekelleher — COMMENTED** [review]
I doesn't explain very much though - maybe link to a wikipedia page about null models or something?

**Review by benjeffery — COMMENTED** [review]
I can't see where JKs comment is addressed?

**Comment by hyanwong** [comment]
Yep - not yet addressed here, as the wikipedia page for Null Models is not very helpful here. Other suggestions welcome.

**Comment by benjeffery** [comment]
Having a bit of a tidy-up - is this still something you'd like to get in @hyanwong?

**Comment by hyanwong** [comment]
Peter's suggestion SGTM. Shall I implement?

**Comment by hyanwong** [comment]
Change made, should be OK to merge.

File: simulation_overview.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ Bayesian Computation). This is especially important in studies of complex demogr
 where there are many potential parameters and models, making it impractical to specify
 likelihood functions.
 
+Compare to expectations
+: It is often useful to compare data to what is expected under a simpler situation
+(e.g. for use as a null model). For instance,  comparison to *neutral* simulations
```
Comment (petrelharp) [inline simulation_overview.md:44]:
```suggestion
(e.g., for use as a null model). For instance,  comparison to *neutral* simulations
```

**Comment by petrelharp** [comment]
LGTM. But, what the heck, CI?

**Pull request merged**